CHANGELOG.md

```diff
@@ -13,11 +13,13 @@
 #### Features
 
 * [#1393](https://github.com/ruby-grape/grape/pull/1393): Middleware can be inserted before or after default Grape middleware - [@ridiculous](https://github.com/ridiculous).
-* [#1390](https://github.com/ruby-grape/grape/pull/1390): Allow inserting middleware at arbitrary points in the middleware stack - [@rosa](https://github.com/rosa).
```

Comment:
If you don't mind cleaning this up. Looks like some case-sensitive changes creeped in here that shouldn't be part of this commit.